odev.js

```diff
@@ -34,3 +34,60 @@ document.querySelector("#reset").addEventListener("click", () => {
 /* START CODING HERE */
 
 
+const firstButton = document.querySelector("#populationBigger")
+firstButton.addEventListener("click", () => {
+    let bt500 = data.filter((city) => city.population > 500000)
+    createTableElements(bt500, "allcities");
+    createTableElements([], "singlecity");
+});
+//second green button
+const secondButton = document.querySelector("#landAreaLess")
+secondButton.addEventListener("click", () => {
+    let lt1000 = data.filter((city) => city.landArea < 1000);
+    createTableElements(lt1000, "allcities");
+    createTableElements([], "singlecity")
+})
+//blue buttons
+
+//isPopulationLess
+const thirdButton = document.querySelector("#isPopulationLess")
+thirdButton.addEventListener("click", () => {
+    let populationLess = data.some((less) => less.population < 100000)
+    if (populationLess) {
+        alert("YES");
+    } else {
+        alert("NO");
+    }
+}
+)
+
+
+//isLandBigger
+const fourthButton = document.querySelector("#isLandBigger")
```

Comment:
Kodu daha okunabilir kılmak için her dom işlemini en üstte tanımlarsan hepsini bir arada görmüş olursun. Bu sayede kod içerisinde gezmek yerine hepsine derli bir şekilde ulaşabilirsin.